lib/web/apiserver.go

```diff
@@ -566,7 +566,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/sites/:site/events/search", h.WithClusterAuth(h.clusterSearchEvents))                 // search site events
 	h.GET("/webapi/sites/:site/events/search/sessions", h.WithClusterAuth(h.clusterSearchSessionEvents)) // search site session events
 	h.GET("/webapi/sites/:site/sessions/:sid/events", h.WithClusterAuth(h.siteSessionEventsGet))         // get recorded session's timing information (from events)
-	h.GET("/webapi/sites/:site/sessions/:sid/stream", h.siteSessionStreamGet)                            // get recorded session's bytes (from events)
+	h.GET("/webapi/sites/:site/sessions/:sid/stream", h.WithClusterAuth(h.siteSessionStreamGet))         // get recorded session's bytes (from events)
```

Comment:
Are you confident that this didn't change the behavior in any way?

Reply:
Good question, no I wasn't. The main difference between this endpoint and the others that get wrapped in `WithClusterAuth` is that with this endpoint, we're sending back a raw octet stream manually, whereas typically we send back a json. 

I've refactored in 27732e7b784525f641eaedab5af3e06816e92398 so that we're keeping all this authentication code centralized (the original intent of the change) while still sending back only a raw octet stream in this handler.

Reply:
27732e7b784525f641eaedab5af3e06816e92398